src/Microsoft.HttpRepl/HttpState.cs

```diff
@@ -34,6 +35,8 @@ public class HttpState
 
         public Dictionary<string, IEnumerable<string>> Headers { get; }
 
+        public Dictionary<string, IEnumerable<string>> QueryString{ get; }
```

Comment:
> Now if the scope of HTTP REPL is just web APIs per Open API

That's certainly a/the focus, but I wouldn't want to constrain us to that when there's a relatively easy way to support more scenarios. 

That said, I see a bit of a problem here in that the list being created here (global query parameters, sent to all subsequent HTTP commands) will be combined with per-command query parameters specified at the time of the specific HTTP command.

It would seem that the best we could do would be to guarantee 
- Order within global query parameters was maintained
- Order within command-specific query parameters was maintained
- Global query parameters are always listed... before? after? command-specific query parameters

I suspect that the situation where order is important AND the user would prefer to have a mixture of global and command-specific parameters would be exceedingly rare, but I wanted to call it out here in case anyone has any thoughts.